packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -53,8 +62,8 @@ type InputBoxComponentProps = {
   inlineChildren: boolean;
   'data-ui-id'?: string;
   id?: string;
-  textValue: string;
-  onChange: (event: FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string | undefined) => void;
+  textValue: string; // This could be plain text or HTML.
```

Comment:
How do we distinguish if this is html or plain text in our code logic?

Reply:
Based on the presence of tags when parsing

Reply:
What if someone intentionally type a text like this:

Hello, today I wanna read book of <Learning C++>

Does the logic strong enough to hold this case